code/modules/admin/verbs/randomverbs.dm

```diff
@@ -1176,7 +1176,7 @@ GLOBAL_LIST_EMPTY(custom_outfits) //Admin created outfits
 		return
 
 	var/list/punishment_list = list(ADMIN_PUNISHMENT_LIGHTNING, ADMIN_PUNISHMENT_BRAINDAMAGE, ADMIN_PUNISHMENT_GIB, ADMIN_PUNISHMENT_BSA)
-
+	var/message = input("Choose the message they will receive") as text

```

Comment:
I was going to say put var/message under var/punishment, but you should actually put var/message under if(QDELETED(target) || !punishment) so it only asks what message if the user has entered a punishment type.